Application/EdFi.Ods.Api/Controllers/VersionController.cs

```diff
@@ -136,6 +136,41 @@ public IActionResult Get()
                                                    "xsd";
                 }
 
+                if (_apiSettings.IsFeatureEnabled(ApiFeature.ChangeQueries.GetConfigKeyName()))
+                {
+                    urlsByName["changeQueries"] = Request.RootUrl(useReverseProxyHeaders) +
+                                                  $"/changeQueries/v{ApiVersionConstants.ChangeQuery}/" +
```

Comment:
This Api Constant and the Identity one below appear to be missing, which is causing the build to fail since the solution cannot be built. Please add these, confirm the build works, and then commit and push the change so it can be re-reviewed.

Reply:
I added the missing constants, reran initdev, reran the tests.  We should be good now.